src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/RequestExtensionCallbackHandlerTest.java

```diff
@@ -178,9 +164,18 @@ void shouldNotUpdateResponseDeadline_whenExtensionIsNotAlreadyAgreed() {
         @Test
         void shouldUpdateBusinessProcess_whenInvoked() {
             CaseData caseData = CaseDataBuilder.builder().atStateExtensionRequested().build();
-            handler.handle(callbackParamsOf(caseData, ABOUT_TO_SUBMIT));
+            CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
 
-            verify(businessProcessService).updateBusinessProcess(new HashMap<>(), REQUEST_EXTENSION);
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("camundaEvent")
+                .isEqualTo(REQUEST_EXTENSION.name());
+
+            assertThat(response.getData())
+                .extracting("businessProcess")
+                .extracting("status")
+                .isEqualTo("READY");
```

Comment:
again I think you can extract multiple / chain here

Reply:
I am afraid not. as we have condition on one property. we can not chain after that.

Reply:
done